services/services.go

```diff
@@ -0,0 +1,84 @@
+// Package services provides methods for managing an app's services.
+package services
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	deis "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+// List services registered with an app.
+func List(c *deis.Client, appID string) (api.Services, error) {
+	u := fmt.Sprintf("/v2/apps/%s/services/", appID)
+	res, reqErr := c.Request("GET", u, nil)
+
+	if reqErr != nil && !deis.IsErrAPIMismatch(reqErr) {
+		return []api.Service{}, reqErr
+	}
+
+	defer res.Body.Close()
+
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return []api.Service{}, err
+	}
+
+	r := make(map[string]interface{})
+	if err = json.Unmarshal([]byte(body), &r); err != nil {
+		return []api.Service{}, err
+	}
+
+	out, err := json.Marshal(r["services"].([]interface{}))
+	if err != nil {
+		return []api.Service{}, err
+	}
+
+	var services []api.Service
+	if err := json.Unmarshal([]byte(out), &services); err != nil {
+		return []api.Service{}, err
+	}
+
+	return services, reqErr
+}
+
+// New adds a service to an app.
```

Comment:
Can you provide a little more detail here for GoDocs? 

https://godoc.org/github.com/teamhephy/controller-sdk-go/apps#New

For example when creating a new app in the docs above, they talk about the newly created random name of the app. Here maybe write a bit more about procfileType and pathPattern that need to be passed to create new service object. Service is related to an application. You can't create a service without having an app first. It's obvious but I think good to be documented.